lora/README.md

```diff
@@ -0,0 +1,75 @@
+# LoRa Adapter
+Adapter between Mainflux IoT system and [LoRa Server](https://github.com/brocaar/loraserver).
+
+This adapter sits between Mainflux and LoRa server and just forwards the messages form one system to another via MQTT protocol, using the adequate MQTT topics and in the good message format (JSON and SenML), i.e. respecting the APIs of both systems.
+
+LoRa Server is used for connectivity layer and data is pushed via this adapter service to Mainflux, where it is persisted and routed to other protocols via Mainflux multi-protocol message broker. Mainflux adds user accounts, application management and security in order to obtain the overall end-to-end LoRa solution.
+
+## Configuration
+
+The service is configured using the environment variables presented in the
+following table. Note that any unset variables will be replaced with their
+default values.
+
+| Variable                         | Description                           | Default               |
+|----------------------------------|---------------------------------------|-----------------------|
+| MF_LORA_ADAPTER_HTTP_PORT        | Service HTTP port                     | 8180                  |
+| MF_LORA_ADAPTER_LOG_LEVEL        | Log level for the Lora Adapter        | error                 |
+| MF_NATS_URL                      | NATS instance URL                     | nats://localhost:4222 |
+| MF_LORA_ADAPTER_LORA_MESSAGE_URL | Loraserver mqtt broker URL            | tcp://localhost:1883  |
+| MF_LORA_ADAPTER_LORA_SERVER_URL  | Loraserver gRPC API URL               | localhost:8080        |
+| MF_LORA_ADAPTER_ROUTEMAP_URL     | Routemap database URL                 | localhost:6379        |
+| MF_LORA_ADAPTER_ROUTEMAP_PASS    | Routemap database password            |                       |
+| MF_LORA_ADAPTER_ROUTEMAP_DB      | Routemap instance that should be used | 0                     |
+| MF_THINGS_ES_URL                 | Things service event store URL        | localhost:6379        |
+| MF_THINGS_ES_PASS                | Things service event store password   |                       |
+| MF_THINGS_ES_DB                  | Things service event store db         | 0                     |
+| MF_LORA_ADAPTER_INSTANCE_NAME    | LoRa adapter instance name            | lora                  |
+
+## Deployment
+
+The service is distributed as Docker container. The following snippet provides
+a compose file template that can be used to deploy the service container locally:
+
+```yaml
+version: "2"
+services:
+  adapter:
+    image: mainflux/lora:[version]
+    container_name: [instance name]
+    environment:
```

Comment:
`MF_LORA_ADAPTER_HTTP_PORT` env variable is missing from here, as well as from the example of the service startup below.